templates/purchaseinvoices.json

```diff
@@ -0,0 +1,153 @@
+[
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-all",
+    "namespaced_identifiers": false,
+    "source": {
+      "operation": "list",
+      "properties": {
+        "endpoint": "purchase_invoices",
+        "primary_key": "id"
```

Comment:
What is this for? I don't see it referenced in the list operation?

Reply:
I see. I think it's not needed there. I duplicated other pipes and edited them. Think I may need to clean up some other data types as well.